oshi-core/src/main/java/oshi/software/os/OSProcess.java

```diff
@@ -76,6 +77,22 @@
      */
     String getCommandLine();
 
+    /**
+     * Gets the command-line arguments of the process.
+     *
+     * <p>On Windows, where the OS models command-line arguments as a single string, this method
+     * computes the approximated tokenization.
+     */
+    List<String> getArguments();
```

Comment:
I'm in favor of empty list, documented as such, unless there's a certain way to distinguish a program with no arguments/environment from one that you just don't have access to read, cross-platform.

> either one may need elevated user privileges for security reasons

Because I expect them to be blank/empty often, when run without elevation, on processes not belonging to the user.